src/arrays/delegate-tasks.js

```diff
@@ -1,177 +1,615 @@
 import Ant from '../images/ant.png';
 import PrimU from '../images/prim-u.png';
 import avatar from '../images/review-avatar.png';
+import graph from '../images/data-layers.png';
+import wordCloud from '../images/word-cloud.png';
+import flumeScience from '../images/flume-science.png';
+import connectedWorld from '../images/connected-world.png';
 
 export const webDevProjects = [
   {
     id: 1,
-    'top-right': {
+    image: {
       image: Ant,
     },
-    'top-left': {
+    info: {
       title: 'Ant Robotics',
       subtitle:
         'Landing page website for the company that cakes automated logistics robots from Oakland, CA',
       date: 'May 2021',
-      link: 'Visit wbesite →',
-    },
-    'bottom-right': {
-      avatar,
-      title: 'Maxim Antonenko',
-      subtitle: 'Ant Robotics CEO',
-      description:
-        '“The Data Analytics students have highlighted a few languages which show a growing demand for translation and expanded our list of keywords to detect more translation-related user reviews,” Demid said. “The Data Science students tested a wide range of natural language processing algorithms, including deep machine learning..."',
-      link: 'Read full review',
-    },
-    'bottom-left': {
-      avatars: [avatar, avatar, avatar],
-      title: 'Students team',
-      subtitle: 'Maria Wright, Maria Wright, Enyel Sequeira',
-      description:
-        '“I learned so much from it,” Xia said. “Not just about new packages and methods of analysis and reporting, but also workflow management, how to clarify task requirements, collaborate with others, and more. Those are the kind of experiences I can put on my resume, and that’s awesome too.”',
+      link: 'https://www.example.com/',
+      linkText: 'Visit website →',
     },
+    reviews: [
+      {
+        id: 11,
+        avatars: [{ id: 1, src: avatar }],
+        title: 'Maxim Antonenko',
+        subtitle: 'Ant Robotics CEO',
+        description:
+          ' “The Data Analytics students have highlighted a few languages which show a growing demand for translation and expanded our list of keywords to detect more translation-related user reviews,” Demid said. “The Data Science students tested a wide range of natural language processing algorithms, including deep machine learning algorithms, to build a robust model that accurately predicts if a review praises, criticizes or requests game translation.”',
+      },
+      {
+        id: 12,
+        avatars: [
+          { id: 1, src: avatar },
```

Comment:
This is great. It's a list of images, so they definitely needed id's so you can map over it. Thanks for catching that